flight/PiOS/Common/pios_mpu.c

```diff
@@ -0,0 +1,1090 @@
+/**
+ ******************************************************************************
+ * @file       pios_mpu.c
```

Comment:
Might rename to pios_mpuxxx0. This supports MPU-XXX0 only so I don't want to name it as if it supports all Invensense families, and I want to follow the existing convention of naming using the model rather than the manufacturer. If it is modified later to support additional device families it's a case of renaming and /s/pios_mpuxxx0/pios_shinynewthingstoo/.